haskell/test/Lorentz/Contracts/Test/FA2.hs

```diff
@@ -246,11 +246,10 @@ fa2Spec fa2Originate = do
 
         withSender commonOperator $ lCallEP stablecoinContract (Call @"Transfer") transfer1
 
-        lExpectStorage stablecoinContract $ \case
-          (StorageLedger ledger)
-            | M.lookup wallet1 ledger == Nothing -> Right ()
-            | otherwise ->
-                Left $ CustomTestError "Zero balance account was not removed"
+        lExpectStorage stablecoinContract $ \storage ->
+          if M.lookup wallet1 (unBigMap $ sLedger storage) == Nothing
```

Comment:
I would use `unless` or `when` here (and in some other places), but that's up to you.

UPD: looks like there are many such places, so probably it's not worth changing them now :)

Reply:
It didn't take long to change, done :)